### ISSUE TITLE
Milestone 2 Checklist

### ISSUE THREAD

**Issue by rtaph** [UBC-MDS/bc_election_turnout#29]
## Version control and submission expectations
- [x] You should be committing to Git every time you work on this project. Thus, we expect to see 10+ commits in your git log by the time you submit this milestone.
- [x] git commit messages should be meaningful. These will be marked. It's OK if one or two are less meaningful, but most should be.
- [x] Each team member is expected to work in their personal fork of the repository, and then frequently send pull-requests of their work to the upstream repo (the one they forked). The other teammates should review, critique (if necessary) and finally accept their teammate's pull request.
- [x] Use GitHub issues to communicate to their team mate (as opposed to email or Slack)
- [x] Just before submitting, create a release on your project repo named 0.1.0.

## Project organization and documentation expectations
- [x] In the main README.md file for this project you should explain the project in a short summary (view from 10,000 feet) and also explain how to run your data analysis (which order scripts are run in, what expected inputs are). Your explanation could include a flow chart/dependecy diagram. This should be an evolution of what was submitted at the proposal stage. Yes, we are asking you to overwrite your proposal here, and so if you want to keep that in a more findable place than in the Git history, do create a new file in the doc directory to archive it there (e.g., proposal.md).
- [x] List the software dependencies for your analysis in the README (or if you use conda, point to the environment.yml file)
- [x] Each individual script must be well documented. This means comments throughout, readable code and a brief summary at the top of each file that answers who wrote it, when it was written and what it does. Additionally, in your project's README.md file, explain how to reproducibly run your scripts from top to bottom, including the arguments that need to be provided to each script.
- [x] Your question, analysis and visualization should make sense. It doesn't have to be complicated, but you should tell a story and have a narrative. Specifically, this should come through in your README and final report file.
- [x] You should use proper grammar and full sentences in your README and report. Point form may occur, but should be less than 10% of your written documents.

## Other Things to Remember
- [x] Re-knit the final doc to PDF and HTML and commit

**Comment by ChadNeald** [comment]
<s>Hey everyone. I checked the example milestone 2 project and it had the following sections so I made this checklist to keep track of what's done and who is working on what. I'm starting on the Data section. I also think the majority of what we have written already can be used for the intro section maybe with just a quick review of it. Feel free to add any sections I've missed!

Completed Sections
- [ ] Summary
- [ ] Introduction
- [ ] Data
- [ ] Analysis
- [ ] Results and Discussion
- [ ] References</s>
